pytests/repo/setup-repo.sh

```diff
@@ -88,7 +88,7 @@ cat << EOF > ${PUBLISH_PATH}/metalink
    <verification>
    </verification>
    <resources maxconnections="1">
-    <url protocol="file" type="file" location="IN" preference="100">file://${PUBLISH_PATH}/repodata/repomd.xml</url>
+    <url protocol="file" type="file" location="IN" preference="100">http://localhost:8080/photon-test/repodata/repomd.xml</url>
```

Comment:
Sid, pls change the protocol and type to http.

Reply:
Done. Thanks!

Reply:
I don't see the changes reflected in the commit.